packages/vscode-extension/src/commands/ImportObjects.ts

```diff
@@ -0,0 +1,225 @@
+				return typeof validationResult === 'string' ? validationResult : null;
+			},
+		});
+		if (appId === undefined) {
+			return;
+		}
+
+		if (appId.length === 0) {
+			appId = defaultAppId;
+		}
+		return appId;
+	}
+
+	private async promptScriptIdFilter() {
+		let scriptId = await window.showInputBox({
+			ignoreFocusOut: true,
+			placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.SCRIPT_ID),
+			validateInput: (fieldValue) => {
+				if (fieldValue === '') {
+					return null; //In this case empty field is valid
+				}
+				let validationResult = InteractiveAnswersValidator.showValidationResults(fieldValue, InteractiveAnswersValidator.validateScriptId);
+				return typeof validationResult === 'string' ? validationResult : null;
+			},
+		});
+
+		return scriptId;
+	}
+
+	private async promptObjectTypes(): Promise<string[] | undefined> {
+		return await window.showQuickPick(
+			objectTypes.map((objectType) => objectType.value.type),
+			{
+				placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.SELECT_TYPES),
+				canPickMany: true,
+			}
+		);
+	}
+
+	private async listObjects(appId: string | undefined, selectedObjectTypes: string[], scriptId: string) {
+		const actionResult = await this.customObjectService.listObjects(appId, selectedObjectTypes, scriptId, this.executionPath);
+		return actionResult;
+	}
+
+	private async promptObjects(objectList: any[]): Promise<string[] | undefined> {
+		const selectedObjects = await window.showQuickPick(
+			objectList.map((object) => object.scriptId),
+			{
+				placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.SELECT_OBJECTS),
+				canPickMany: true,
+			}
+		);
+		return selectedObjects;
+	}
+
+	private async promptIncludeReferencedFiles() {
+		return await window.showQuickPick([this.translationService.getMessage(ANSWERS.YES), this.translationService.getMessage(ANSWERS.NO)], {
+			placeHolder: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.REFERENCED_FILES),
+			canPickMany: false,
+		});
+	}
+
+	private async promptOverwrite(includeReferencedFiles: string | undefined) {
+		return await window.showQuickPick([this.translationService.getMessage(ANSWERS.YES), this.translationService.getMessage(ANSWERS.NO)], {
+			placeHolder:
+				includeReferencedFiles === this.translationService.getMessage(ANSWERS.NO)
+					? this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.OVERWRITE_WITH_REFERENCED)
+					: this.translationService.getMessage(IMPORT_OBJECTS.QUESTIONS.OVERWRITE),
+			canPickMany: false,
+		});
+	}
+
+	private showOutput(actionResult: any) {
+		if (actionResult.status === actionResultStatus.SUCCESS && actionResult.data) {
+			if (actionResult.data.length === 0) {
+				this.messageService.showCommandError(this.translationService.getMessage(IMPORT_OBJECTS.ERROR.EMPTY_LIST_SEARCH));
+				return;
```

Comment:
When we reach this point of execution because there were no objects to list (with the criteria previously selected), we show the `see details` button, but there is nothing in the SuiteCloud output to be shown.